src/popup/pages/MultisigProposalDetails.vue

```diff
@@ -595,7 +602,7 @@ export default defineComponent({
   }
 
   .bottom-buttons {
-    padding: 24px;
+    padding: var(--screen-padding-x);
```

Comment:
The variable value and the previous value of the padding are different. Are you sure with this changes?

Reply:
That's what SW-828 was about - to align the padding of the buttons with the view